client/my-sites/hosting/web-server-settings-card/index.js

```diff
@@ -113,6 +113,12 @@ const WebServerSettingsCard = ( {
 				} ),
 				value: '8.1',
 			},
+			{
+				label: translate( '8.2', {
```

Comment:
Do these numbers necessarily need to be translated? I'd just expect no translate for plain numbers nad something like this for additional labels:

```js
translate( '%s (experimental)', { args: '8.2' } )
```

Reply:
I think I added them initially for `rtl`, but maybe not needed. Could ask someone from i18n? Tho easier to change when adding these flags.

Reply:
cc @Automattic/i18n for advice on above

Reply:
I'd recommend localizing the number, but [number_format_i18n](https://developer.wordpress.org/reference/functions/number_format_i18n/) should be a better way of doing that.

Reply:
For Calypso that'd be [numberFormat](https://github.com/Automattic/wp-calypso/tree/trunk/packages/i18n-calypso#numberformat-method) 🙂

Reply:
Hmm, these are version numbers, not floats. Probably easier to leave as is, since there's a chance it might be 8.2.1 at some point, and keep it the same across all versions?

Reply:
Right, I get translating floats because the comma notation changes across locales but software versions don't change between locales (as far as I know), and nobody would ever [translate the "8.2"](https://translate.wordpress.com/deliverables/overview/7663272) to anything else anyway? We'd just end up re-sending "recommended" string for re-translating when versions change.

LTR/RTL would work via CSS in this case I'd assume.

Reply:
Deal. Will remove the 2 unneeded, amd switch to your recommendation.